x-pack/legacy/plugins/siem/public/pages/detection_engine/components/signals/signals_utility_bar/index.tsx

```diff
@@ -98,11 +78,13 @@ const SignalsUtilityBarComponent: React.FC<SignalsUtilityBarProps> = ({
                 </UtilityBarText>
 
                 <UtilityBarAction
-                  iconSide="right"
-                  iconType="arrowDown"
-                  popoverContent={getBatchItemsPopoverContent}
+                  disabled={areEventsLoading || Object.keys(selectedEventIds).length === 0}
```

Comment:
isEmpty would work well here if that's your cup of tea 🍵 

```ts
_.isEmpty({})
true
```

Reply:
you know it is